packages/shared/src/lib/func.ts

```diff
@@ -63,3 +63,11 @@ export enum ArrowKeyEnum {
   Down = 'ArrowDown',
   Left = 'ArrowLeft',
 }
+
+export const shuffle = (array: Array<unknown>): void => {
```

Comment:
We could actually use `_.shuffle` to avoid building logic ourselves?

Reply:
In any case the method should return new array without mutating in place.

Reply:
Yeah might as well